provisioning/service/samples/ProvisioningServiceEnrollmentGroup/Program.cs

```diff
@@ -97,6 +101,24 @@ public static void Main(string[] args)
             }
         }
 
+        /// <summary>
+        /// Enumerates all registrations within an enrollment group.
+        /// </summary>
+        /// <param name="provisioningServiceClient">The ProvisioningServiceClient object.</param>
+        /// <param name="querySpecification">The query specification.</param>
+        /// <param name="group">The EnrollmentGroup object.</param>
+        /// <returns></returns>
+        private static async Task EnumerateDevicesInGroup(ProvisioningServiceClient provisioningServiceClient, QuerySpecification querySpecification, EnrollmentGroup group)
+        {
+            Console.WriteLine($"\nCreating a query for registrations within group '{group.EnrollmentGroupId}'...");
+            using (Query registrationQuery = provisioningServiceClient.CreateEnrollmentGroupRegistrationStateQuery(querySpecification, group.EnrollmentGroupId))
```

Comment:
Would be nice to have an example query specification so it's clear what the user needs to construct.

Reply:
@nberdy - we're passing the same one. I thought the service doesn't support anything else here.
/cc @alextolp

Reply:
Ignore me, I missed the definition because it was collapsed in the PR